standalone/test/unit_tests/util/unittest_tenth_of_degree.cpp

```diff
@@ -135,9 +135,7 @@ TEST(TenthOfDegreeTest, LargerThanOrEqualComparison)
 
 TEST(TenthOfDegreeTest, shouldThrowUnderflowErrorOnCastToIntOfValuesUnderZero)
 {
-  EXPECT_THROW(std::cout << static_cast<uint16_t>(util::TenthOfDegree(-1))
-                         << " - printed to not be unused. Should throw before!\n",
-               std::underflow_error);
+  EXPECT_THROW(static_cast<uint16_t>(util::TenthOfDegree(-1)), std::underflow_error) << "Should throw underflow error";
```

Comment:
This should solve the issue
```suggestion
  uint16_t result __attribute__((unused));
  EXPECT_THROW(result = static_cast<uint16_t>(util::TenthOfDegree(-1)), std::underflow_error)
      << "Should throw underflow error";
```